app/assets/javascripts/components/event_form.js.jsx

```diff
@@ -51,6 +56,12 @@ var ScheduleEventForm = React.createClass({
               value={this.state.Description}
               className={this.state.className}/>
 
+            <label htmlFor='Start'>Date and Time</label>
+            <input id='datetime' type="datetime"
```

Comment:
The `htmlFor` name needs to match the `id` of the input that it is a label for. That is what connects them.